src/components/Schedule.vue

```diff
@@ -96,6 +87,8 @@ export default {
 
       const current = new Date();
 
+      this.currentWeek = null
```

Comment:
unnecessary code. Default is value `undefined` already.

Reply:
The default value was not undefined, so I have set it to undefined.